marvin-ai/marvin-automl-ui
# Nvl issue 13

To test this PR:

1. Setup the environment;
2. Setup the AutoML Engine
3. Run all steps with AutoML Engine example data
4. See the prediction
